core/types/transaction.go

```diff
@@ -581,6 +581,10 @@ func (tx *Transaction) SetPrivate() {
 	}
 }
 
+func (tx *Transaction) IsPrivacyMarker() bool {
```

Comment:
```suggestion
func (tx *Transaction) HasPrivacyMarker() bool {
```
many "is privacy marker" changes, I prefer with has, or use

Reply:
This hasn't changed since the PR was originally reviewed. We don't want to make further changes at this late stage now.